src/Components/AppConfig/AppConfig.tsx

```diff
@@ -105,6 +116,34 @@ const AppConfig = ({ plugin }: Props) => {
           />
         </Field>
 
+        <Field
+          className={styles.marginTop}
+          description={
+            <span>
+              Disables Logs Drilldown&apos;s usage of the{' '}
+              <a
+                className="external-link"
+                href="https://grafana.com/docs/loki/latest/reference/loki-http-api/#patterns-detection"
+                target="_blank"
+                rel="noreferrer"
+              >
+                Loki Patterns API
+              </a>{' '}
+              endpoint
```

Comment:
"and disables the Patterns tab" or something like that, because both things happen.

Reply:
Also something about requiring a page refresh/reload.

Reply:
Not a blocker, but I'd love to avoid questions like "I disabled patterns but I still see them".

Reply:
They don't know what's good 😎